pages/en/lb4/readmes/example-getting-started.md

```diff
@@ -0,0 +1,424 @@
+## Making A Todo App
```

Comment:
I believe the files in `readmes` are automatically fetched from GitHub, see http://loopback.io/doc/en/contrib/Including-READMEs.html and https://github.com/strongloop/loopback.io/blob/e26a44be2e910740a4eaea2927845b46812a2fd1/update-v4-readmes.sh.

IMO, you need to make these changes in our main monorepo - https://github.com/strongloop/loopback-next/blob/a2368cee598c131a826ee42e347266c6e25ae582/packages/example-getting-started/README.md

If you don't want this content to be fetched from example-getting-started's README, then please put the content to a different file.

Unless I am missing something here?

Reply:
I'm not exactly sure how it's supposed to work, but I know that the current template we use for transplanting the READMEs into the documents is currently hardcoded to path towards `strongloop/{repo-name}`, which won't work for our use-case. It looks like we're refreshing those doc pages by _committing the updates here_ rather than scraping them dynamically, since we have a whole folder full of READMEs for those connectors in the loopback.io repository: https://github.com/strongloop/loopback.io/tree/gh-pages/pages/en/lb3/readmes

Sidenote: It also looks like many of them haven't been updated in forever; was this script run manually by Rand or someone else on a regular basis?

Regardless, I wouldn't expect there to already be something in place that copies READMEs over from lb4, and until there is, we need to make sure that there's something in place here.

Reply:
Quick addendum: I do think that figuring out an automatic refresh of these documents falls within MVP scope, but only if everyone agrees:

@bajtos @raymondfeng @dhmlau

Reply:
I remember some of the READMEs in LB3 got copied over to the official documentation.  Not sure how easy/difficult to do that.  Perhaps @bschrammIBM can help us?

Reply:
We are already fetching some READMEs from loopback-next repository, see https://github.com/strongloop/loopback.io/pull/588 for an example how to do that, and also Raymond's recent PR https://github.com/strongloop/loopback.io/pull/608 that's fixing some aspects of this process.

> Sidenote: It also looks like many of them haven't been updated in forever; was this script run manually by Rand or someone else on a regular basis?

I believe there is a Jenkins pipeline (see [Jenkinsfile](https://github.com/strongloop/loopback.io/blob/gh-pages/Jenkinsfile) in this repo) that is supposed to update README whenever they change. I think it's based on some GitHub web hooks, but I am not sure. @rmg could you please clarify?

> Regardless, I wouldn't expect there to already be something in place that copies READMEs over from lb4, and until there is, we need to make sure that there's something in place here.

Uff, I am don't understand, could you please rephrase your sentence for more clarity?

Since we already have some tooling for including READMEs from loopback-next packages, I think we should leverage it for your work too, even if this process has flaws right now. At worst, you can run [update-v4-readmes.sh](https://github.com/strongloop/loopback.io/blob/gh-pages/update-v4-readmes.sh) manually to ensure the copy of example-getting-started's README is stored in this repo.

If you disagree and think this is more work that we want to invest for MVP, then please apply the second approach I proposed:

> If you don't want this content to be fetched from example-getting-started's README, then please put the content to a different file.

I really don't want to create a situation when it looks like `pages/en/lb4/readmes/example-getting-started.md` is something that's supposed to be automatically mirroring content of `example-getting-started/README.md`, while it is not a mirror but a completely different content instead.